Makefile

```diff
@@ -89,47 +87,62 @@ ifneq ($(CHECKSTYLE),0)
 	PERL5LIB=lib/perlcritic:$$PERL5LIB perlcritic lib
 endif
 
-.PHONY: test
-ifeq ($(TRAVIS),true)
-test: run-tests-within-container
-else
-test: checkstyle test-with-database
-endif
-
-.PHONY: test-unit-and-integration
-test-unit-and-integration:
-	OPENQA_CONFIG= prove ${PROVE_LIB_ARGS} ${PROVE_ARGS}
+test-with-db%: 
+	@$(MAKE) $(subst test-with-db,test-with-database,$@)
 
-.PHONY: test-with-database
-test-with-database:
+test-with-db:
+	@$(MAKE) test-with-database   
+ 
+test-with-databas%:
```

Comment:
shouldn't that be "databas**e**"?

Also, can you still declare these .PHONY? I doubt this is implicit for template targets